app/views/admin/lead_provider_users/edit.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= form_for @user, url: admin_lead_provider_user_path do |f| %>
+  <%= render "form_fields", locals={f: f} %>
+  <%= f.govuk_submit "Update" %>
+<% end %>
+
+<%= govuk_back_link text: "Back", href: admin_lead_provider_users_path(@lead_provider) %>
```

Comment:
Nitpick, but I am not sure we want back links to be at the bottom of a page - top of the page seems to be a more common pattern. Presumably so you don't need to scroll down to find it on long pages, and to keep it consistent with them on short pages.

Reply:
Ah yeah, force of habit. NHS style guide is at the bottom of the page, their research showed that screenreader users found them distracting at the top of the page. I'll move them to the top. @cfq any views on this?

Reply:
Maybe we want to use [the back link](https://design-system.service.gov.uk/components/back-link/)? 

Which says about itself:
> Always place back links at the top of a page, before the <main> element. Placing them here means that the “Skip to main content” link allows the user to skip all navigation links, including the back link.

Reply:
Yeah, that's what I'm using, just put it in the wrong place. I've internalised the NHS design system, which reads

> The GOV.UK Design System says that you should put back links at the top of a page, but a couple of NHS services (NHS 111 online and Find out why your NHS data matters) found that it works better at the bottom of the page below the primary action. That's because we don't want to suggest to people who use a screen reader that they leave the page prematurely.

Reply:
https://service-manual.nhs.uk/design-system/components/back-link

Reply:
Ah, my bad, for some reason I thought it was a button 😕